### PR TITLE
Fixes after sendgrid package update. Add stats email address(es) to c…

### PR DESCRIPTION
**Pull request by greywidget**
…onfig

*** Please modify your .env file with the new config value e.g:

PYBYTES_EMAIL_GROUP=me julian@pybit.es bob@pybit.es

### CODE REVIEW

File: .env-template [hunk 1]
```diff
@@ -10,3 +10,4 @@ FROM_EMAIL=
 DOMAIN=http://localhost:8000
 ENV=local
 SLACK_VERIFICATION_TOKEN=
+PYBITES_EMAIL_GROUP=bob@pybit.es julian@pybit.es
```
Comment (bbelderbos) [inline .env-template:13]: @greywidget minor thing, can you please make this a comma separated list, here is an example how we use it for allowed hosts:

```
.env
ALLOWED_HOSTS=.localhost, .herokuapp.com

settings.py
ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())
```

File: myreadinglist/mail.py [hunk 2]
```diff
@@ -2,43 +2,41 @@
...
-    body = body.replace('\n', '<br>')
+    from_email = From(email=from_email, name=PYBITES)
+    to_email = To(to_email)
 
     # if local no emails
     if settings.LOCAL:
+        body = body.replace('<br>', '\n')
         print('local env - no email, only print send_email args:')
-        print('to_email: {}'.format(to_email))
+        print('to_email: {}'.format(to_email.email))
```
Comment (bbelderbos) [inline myreadinglist/mail.py:21]: could turn those into f-strings actually

File: myreadinglist/mail.py [hunk 3]
```diff
@@ -47,3 +45,7 @@ def send_email(to_email, subject, body, from_email=FROM_EMAIL, html=True):
         )
 
     return response
+
+
+if __name__ == '__main__':
+    send_email('test-email@gmail.com', 'my subject', 'my message')
```
Comment (bbelderbos) [inline myreadinglist/mail.py:51]: just for ideas, here you could add `argparse` so we can call it as standalone script with say `-s subject -m message`, but leave as is, we don't need that for now

File: myreadinglist/management/commands/stats.py [hunk 4]
```diff
@@ -72,5 +74,5 @@ def handle(self, *args, **options):
                          books_completed=books_completed,
                          goals=goals_out)
 
-        for to_email in 'me julian@pybit.es bob@pybit.es'.split():
+        for to_email in PYBITES_EMAIL_GROUP.split():
```
Comment (bbelderbos) [inline myreadinglist/management/commands/stats.py:77]: no split needed if we change it to `config('PYBITES_EMAIL_GROUP', cast=Csv())` on line 12 (see comment above)

**Review by bbelderbos — CHANGES_REQUESTED** [review]
Small change please then we'll be good. 
Thanks, 
Bob

File: myreadinglist/mail.py (hunk 2 above)
Comment (greywidget) [reply myreadinglist/mail.py]: done

File: myreadinglist/mail.py (hunk 3 above)
Comment (greywidget) [reply myreadinglist/mail.py:49]: Cool, happy to do this anytime

**Comment by greywidget** [comment]
All requested changes should be done now Bob

**Comment by bbelderbos** [comment]
@greywidget perfect, thank you!

**Review by bbelderbos — APPROVED** [review]
Awesome, thanks

**Pull request merged**